x/wasm/keeper/query_plugins_test.go

```diff
@@ -763,6 +764,7 @@ func TestAcceptListStargateQuerier(t *testing.T) {
 }
 
 func TestDistributionQuerier(t *testing.T) {
```

Comment:
Do you think this test can be deleted?

Reply:
ja, I will remove the test. Thanks for finding this!